src/Tests/Tests/ClientConcepts/ConnectionPooling/BuildingBlocks/ConnectionPooling.doc.cs

```diff
@@ -79,6 +82,88 @@ public class ConnectionPooling
 			client.ConnectionSettings.ConnectionPool
 				.Should().BeOfType<SingleNodeConnectionPool>();
 		}
+		
+		/**
+		* [[cloud-connection-pool]]
+		* ==== CloudConnectionPool
+		*
+		* A specialized subclass of `SingleNodeConnectionPool that accepts a Cloud Id and credentials.
```

Comment:
link to Cloud docs for Cloud Id

Reply:
I can't use `{ref_current}` for https://www.elastic.co/guide/en/cloud/current/ec-cloud-id.html would need to add a new block I think. 

Is my assumption correct @russcam ?